pkg/dynamicinstrumentation/ebpf/ebpf.go

```diff
@@ -53,42 +54,29 @@ func AttachBPFUprobe(procInfo *ditypes.ProcessInfo, probe *ditypes.Probe) error
 		return fmt.Errorf("could not create bpf collection for probe %s: %w", probe.ID, err)
 	}
 
+	outerMapSpec := spec.Maps["param_stacks"]
+	outerMapSpec.MaxEntries = uint32(runtime.NumCPU())
```

Comment:
Please use `kernel.PossibleCPUs()`, `runtime.NumCPU()` is not guaranteed to match the kernel.